bin/runtime/src/lib.rs

```diff
@@ -219,10 +218,21 @@ impl pallet_aura::Config for Runtime {
     type DisabledValidators = ();
 }
 
+parameter_types! {
+    pub const UncleGenerations: BlockNumber = 5;
```

Comment:
Wait, what the hell is that :P

Reply:
I just finished my review in which I had a similarly perplexed reaction. :P